modules/node/handler/cms_pagination_builder.e

```diff
@@ -0,0 +1,67 @@
+note
+	description: "Generic Pagination Builder Interface"
+	date: "$Date$"
+	revision: "$Revision$"
+
+deferred class
+	CMS_PAGINATION_BUILDER [G]
+
+feature -- Access
```

Comment:
very minor

clause note should be -- Element change
and usually the -- Access comes first, and then the -- Element change